src/api/telemetry/LegacyTelemetryProvider.js

```diff
@@ -141,17 +141,20 @@ define([
         return capability.subscribe(callbackWrapper, request);
     };
 
-    LegacyTelemetryProvider.prototype.limitEvaluator = function (domainObject) {
+    LegacyTelemetryProvider.prototype.supportsLimits = function (domainObject) {
         var oldObject = this.instantiate(
             utils.toOldFormat(domainObject),
-            utils.makeKeyString(domainObject.identifier));
-        var limitEvaluator = oldObject.getCapability("limit");
+            utils.makeKeyString(domainObject.identifier)
+        );
+        return oldObject.hasCapability("limit");
+    };
 
-        if (!limitEvaluator) {
-            return {
-                evaluate: function () {}
-            };
-        }
+    LegacyTelemetryProvider.prototype.getLimitEvaluator = function (domainObject) {
```

Comment:
I wonder if there will be a case for a default limit evaluator that reads limits from telemetry metadata? This would make it simple for folks to get up and running with limits, without having to write yet another provider.

Reply:
See [comment](https://github.com/nasa/openmct/pull/1941#pullrequestreview-105464455) above, not going to add a default at this time due to this being in flux.